addons/mail/static/src/new/discuss/autogrow_input.js

```diff
@@ -0,0 +1,97 @@
+/** @odoo-module */
+
+import { useFocus, useHover } from "../utils";
+import {
+    Component,
+    useRef,
+    useState,
+    useExternalListener,
+    onWillUpdateProps,
+} from "@odoo/owl";
```

Comment:
Prettier says should be single line.

Make sure you have properly set up `@web/tooling`. Normally whenever you do `git commit`, it should apply prettier on all files in diff, so this LOC should have been re-formatted according